packages/contracts/test/diamond/facets/CurveDollarIncentiveFacet.t.sol

```diff
@@ -95,7 +89,7 @@ contract CurveDollarIncentiveTest is DiamondSetup {
         ICurveDollarIncentiveFacet.incentivize(
             stableSwapPoolAddress,
             mockReceiver,
-            amountIn
+            100e18
```

Comment:
let's use amountIn and make it an input parameter to make this test a fuzzing one

Reply:
Alight. Updating.